### ISSUE TITLE
Documentation not working

### ISSUE THREAD

**Issue by artpelling** [emacs-lsp/lsp-pyright#68]
Hello,

I have the problem that `lsp-ui-show-doc` is very short and `company-show-doc-buffer` just opens an empty buffer for many functions (bultins and numpy for example). I tried many things and I also manually installed type stubs that are found by pyright but nothing helps.
Is anyone having this problems as well? Looking for any pointers on how to fix this.

When I show the doc buffer with `anaconda-mode` I get correct results, so I doubt it is my Python installation.

**Comment by mgpanpan** [comment]
Have you solved this problem? I have the similar problem, my `lsp-ui-show-doc` only shows the function signature, not the whole document.

**Comment by artpelling** [comment]
> Have you solved this problem? I have the similar problem, my `lsp-ui-show-doc` only shows the function signature, not the whole document.

Nope, still have this issue and still using `anaconda-mode` which actually is fine.

**Comment by sasinhe** [comment]
Has this issue been solved already? I'm experiencing the same thing.

**Comment by Swarsel** [comment]
Has anyone found a solution to this problem?

**Comment by artpelling** [comment]
> Has anyone found a solution to this problem?

So far, I am not aware of any solution. I am still using the browser for docs 🥺
